### DISCUSSION TITLE
Finding X and Y coordinates

### DISCUSSION THREAD

**Discussion by Brouwertje** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#289] [Q&A] [answered]
I just can't find coordinates and don't know what i'm missing. I select Pin & go on the map card, choose a spot and double click the go button. (keep the button pressed does nothing) I then see a popup that says ''Configuration available in browser's console'' 

I check console, but can't find the coordinates. Where do i have to look exactly, does anyone know?

Map is working correctly and all, so that is not the issue. If I press the go button once, my roborock s7 moves like a charm.

**Comment by PiotrMachowski** [comment, marked as answer]
You have to long press it, not double click. If long press doesn't work you can add `debug: true` in card's config and just normally click "play".

**Comment by Brouwertje** [comment]
Hmm, I was using debug wrong I guess, now it's working! Thank you! Also for the great work on this project, very nice.
